portal/client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
@kevalnagda @chsasank 
Important for the OSS project to have proper README to get started on the project, lets remove this default from CRA and use our own proper README

Reply:
Sure, will update this.